doc/rst/source/movie.rst

```diff
@@ -218,7 +218,8 @@ Optional Arguments
 **-M**\ [*frame*],[*format*]
```

Comment:
```suggestion
**-M**\ [*frame*\|\ **f**\|\ **m**\|\ **l**],[*format*]
```

Reply:
Including the codes here and in the synopsis (L28) might make the new option more widely known.